org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/client/ExtendedClientCapabilities.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4xml.client;
+
+/**
+ * Extended client capabilities which is not supported by LSP.

```

Comment:
Extended client capabilities not defined by the LSP.

Reply:
@fbricon updated